Tests/test_NCBIXML.py

```diff
@@ -1865,7 +1867,21 @@ def test_xml_2218L_rpsblast_001(self):
         # <Iteration_message>CONVERGED</Iteration_message>
         self.assertRaises(StopIteration, next, records)
         handle.close()
-
+    
+    def test_database_letters_deprecation(self):
+        with warnings.catch_warnings():
+            warnings.simplefilter("error", BiopythonDeprecationWarning)
+            filename = 'xml_2212L_blastp_001.xml'
+            with open(os.path.join("Blast", filename)) as handle:
+                try:
+                    records = NCBIXML.parse(handle)
+                except BiopythonDeprecationWarning as de:
```

Comment:
Another typo ``de`` but in the next line you use ``e`` (short for error or exception).

PEP8 says using single letter variables is consider poor style, but ``e`` is commonly quite used in this situation.